File: 신고결과받기/javascript/20221108/solution.test.js
import solution from './solution';

test('solution 결과', () => {
  const id_list = ['muzi', 'frodo', 'apeach', 'neo'];
  const report = ['muzi frodo', 'apeach frodo', 'frodo neo', 'muzi neo', 'apeach muzi'];
  const k = 2;
  expect(solution(id_list, report, k)).toStrictEqual([2, 1, 1, 0]);
});

File: programmers/level1/신고결과받기/java/20221107/test/SolutionTest.java
import org.junit.jupiter.api.Test;

import static org.junit.jupiter.api.Assertions.assertArrayEquals;

class SolutionTest {

    @Test
    void solution() {
        Solution solution = new Solution();

        String[] id_list = {"muzi", "frodo", "apeach", "neo"};
        String[] report = {"muzi frodo", "apeach frodo", "frodo neo", "muzi neo", "apeach muzi"};
        int k = 2;

        assertArrayEquals(new int[]{2, 1, 1, 0}, solution.solution(id_list, report, k));
    }

}
